frontend/src/store/slices/users/users.slice.ts

```diff
@@ -0,0 +1,38 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
+import { ReducerName } from 'common/enums';
+import { AppThunk } from 'common/types';
+import { userApi } from 'services';
+import { IUserTypeDoctor } from 'components/doctors-search/common/interfaces'
+
+type DoctorsState = {
+  doctors: IUserTypeDoctor[]
+  loading: boolean
```

Comment:
Instead of flag use the `DataStatus`
https://github.com/BinaryStudioAcademy/bsa-2021-healthcare/pull/4/files#diff-6026d4b2eb8ffa36dafc311b054e2976c23a2e952e5d62553c83e34527ac0d2a